i18n/locales/en.json

```diff
@@ -192,7 +187,27 @@
       }
     },
     "blogIndex": {
-      "currentYear": "Blog from {year}"
+      "categoryName": "{category, select, all {Blog} other {{category} Blog Posts}}"
+    },
+    "blog": {
+      "title": "Blog",
+      "subtitle": "The latest Node.js news, case studies, tutorials, and resources.",
+      "selectCategory": "Categories",
+      "categories": {
+        "all": "Everything",
+        "announcements": "Announcements",
+        "release": "Releases",
+        "vulnerability": "Vulnerabilities",
+        "advisory-board": "Advisory Board",
+        "community": "Community",
+        "feature": "Feature",
+        "module": "Module",
+        "npm": "NPM",
```

Comment:
npm should not be capitalized (it's [not an acronym](https://docs.npmjs.com/policies/logos-and-usage#general-rules)).